REQUIRE

```diff
@@ -1 +1 @@
-julia 0.4
+julia 0.5-
```

Comment:
Oh, okay. Then the Compat stuff I've mentioned in here should be unnecessary. If you only want to support 0.5 and up in version 0.9.0 then there should be no user-facing changes on Julia 0.4 since `Pkg.update()` simply won't upgrade TimeSeries to 0.9.0.

IMHO once you drop support for 0.4 you shouldn't look back. (By that I mean in a subsequent release readd support for 0.4 after it was dropped in a prior release)